metron-analytics/metron-profiler/README.md

```diff
@@ -137,15 +137,20 @@ This section will describe the steps required to get your first "Hello, World!""
     hbase(main):001:0> count 'profiler'
     ```
 
-1. Use the Profiler Client to read the profile data.  The below example `PROFILE_GET` command will read data written by the sample profile given above, if 10.0.0.1 is one of the input values for `ip_src_addr`.
-More information on configuring and using the client can be found [here](../metron-profiler-client).
-It is assumed that the `PROFILE_GET` client is correctly configured before using it.
+1. Use the [Profiler Client]((../metron-profiler-client)) to read the profile data.  The following `PROFILE_GET` command will read the data written by the "Hello, World" profile. This assumes that `10.0.0.1` is one of the values for `ip_src_addr` contained within the telemetry consumed by the Profiler.
```

Comment:
Hello, world vs hello-world? Should the naming be consistent here to avoid confusion?